.jenkins/jenkins_test1.sh

```diff
@@ -1,13 +1,14 @@
 #!/bin/bash
 
-# Script for Jenkins continuous integration testing of theano base and old backend
+# Script for Jenkins continuous integration testing of theano base
 
-echo "===== Testing theano base"
+echo "===== Testing theano core"
 
 # Get environment from worker, necessary for CUDA
 source ~/.bashrc
 
-# Test theano CPU and old GPU backend sandbox.cuda
-THEANO_PARAM="theano --with-timer --timer-top-n 10"
-FLAGS="mode=FAST_RUN,init_gpu_device=gpu"
+# Test theano core
+PARTS="theano/compat theano/compile theano/d3viz theano/gof theano/misc theano/sandbox/linalg theano/sandbox/tests theano/scalar theano/scan_module theano/sparse theano/tensor theano/tests theano/typed_list"
```

Comment:
I don't like this white list. If we add one directory, it won't get executed.

Can you use the -e parameter to exclude directory? We use it in the .travis.yml file. This will prevent futur error.